python/src/controllers/opgaver_controller.py

```diff
@@ -0,0 +1,184 @@
+from flask import request, jsonify
+from datetime import datetime
+from models import Opgaver
+from utils.db_connection import get_db_client
+
+db_client = get_db_client()
+
+
+def create_opgaver_with_forloebsskabelon_id():
+    session = db_client.get_session()
+    try:
+        data = request.json
+        if 'ForløbsskabelonID' not in data:
+            return jsonify({"error": "Missing required field: ForløbsskabelonID"}), 400
+
+        opgave = Opgaver(
+            title=data['title'],
+            beskrivelse=data['beskrivelse'],
+            resourcer=data['resourcer'],
+            ansvarlig=data['ansvarlig'],
+            startdato=datetime.fromisoformat(data['startdato']),
+            slutdato=datetime.fromisoformat(data['slutdato']),
+            result=data['result'],
+            timestamp=datetime.fromisoformat(data['timestamp']),
+            ForløbsskabelonID=data['ForløbsskabelonID']
+        )
+        session.add(opgave)
+        session.commit()
+        return jsonify({"message": "Opgaver created successfully with ForløbsskabelonID"}), 201
+    except Exception as e:
+        session.rollback()
+        return jsonify({"error": str(e)}), 500
+    finally:
+        session.close()
+
+
+def create_opgaver_with_forloeb_id():
+    session = db_client.get_session()
+    try:
+        data = request.json
+        if 'ForløbID' not in data:
+            return jsonify({"error": "Missing required field: ForløbID"}), 400
+
+        opgave = Opgaver(
+            title=data['title'],
+            beskrivelse=data['beskrivelse'],
+            resourcer=data['resourcer'],
+            ansvarlig=data['ansvarlig'],
+            startdato=datetime.fromisoformat(data['startdato']),
+            slutdato=datetime.fromisoformat(data['slutdato']),
+            result=data['result'],
+            timestamp=datetime.fromisoformat(data['timestamp']),
+            ForløbID=data['ForløbID']
+        )
+        session.add(opgave)
+        session.commit()
+        return jsonify({"message": "Opgaver created successfully with ForløbID"}), 201
+    except Exception as e:
+        session.rollback()
+        return jsonify({"error": str(e)}), 500
+    finally:
+        session.close()
+
+
+def get_all_opgaver():
```

Comment:
Ændr til get_all_opgaveskabeloner. Vi har ikke brug for at trække alle opgaver på én gang, kun alle opgaver med specifikt forloebsID